### DISCUSSION TITLE
Problema em adicionar permissao e usuario

### DISCUSSION THREAD

**Discussion by rato0** [RamonSilva20/mapos#2340] [General]
estava com problema em adicionar permissao e em adicionar usuario na versao nova baixada aqui, vi que tinham alterado recentemente o arquivo config.php e coloquei o config.php de uma versao anterior e voltou ao normal.

**Comment by brunolima23** [comment]
opa,  estou  com esse problema tbm, vc colocou a config de qual  versao?

**Reply by rato0** [reply]
Desculpa mas n sei a versao, eh porq tinha baixado direto de um video e estava funcionando normal, ai atualizei pra versao do repositorio e deu isso, ai fiz esse teste e deu certo mas exclui os arquivos antigos

**Comment by fapernas** [comment]
Olá, estou na versão 4.44 instalada do zero. Até agora às funções que utilizei estão ok, mas ao tentar criar um novo usuário tomo a mensagem 'The action you have requested is not allowed'. @rato0 por acaso era essa a mensagem pra você? De que versão você pegou o config.php?
Resolveu esse problema de fato? Deu alguma 'incompatibilidade' ou outros erros?

Valeu!!

**Reply by rato0** [reply]
Essa mensagem msm, o restante funcionou normal, mas sugiro que faca backup do arquivo para se previnir, nao vou saber qual a versao pois exclui os outros arquivos

**Reply by pabloruancoder** [reply]
então compartilha o config.php homi
